### ISSUE TITLE
Co-ordinate and prep for Accessibility Audit

### ISSUE THREAD

**Issue by NNavickas1** [ministryofjustice/data-platform#593]
## User Story

As a delivery manager
I want to coordinate the accessibility audit for my service 
So that we can ensure our service meets WCAG2.2

### Acceptance Criteria

- [ ] audit organised 
- [ ] audit prepared for, per [this](https://justiceuk.sharepoint.com/sites/knowthething/SitePages/Get-an-accessibility-audit.aspx#prepare-for-your-audit) 

> You are responsible for defining what the audit should cover, including preparing a brief for the audit.
> 
> You must ensure that the audit supplier can access your service, including any test environments. Audits cannot be quoted or completed if access is not available.

> It’s usually not feasible to audit the whole service. You should focus on:
> 
> - key user journeys
> - representative page types
> - key interactive features
> - any known problem areas
> - all relevant parts of your service (for example, public-facing and admin areas)
> - anything custom-built
> 

### Description/Context

Following tranche 2 of work for the AI Gateway, there is sufficient capability enabled on the service, to instigate an Accessibility Audit. These audits cost north of £7k, so it's pertinent to schedule them when there is sufficient capability.

### Definition of Done

- [ ] audit scheduled 
- [ ] team has prepared necessary information to enable the audit
